dbclient/hikari/src/main/java/module-info.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import io.helidon.common.features.api.Feature;
+import io.helidon.common.features.api.HelidonFlavor;
+import io.helidon.dbclient.hikari.HikariConnectionPoolProvider;
+import io.helidon.dbclient.jdbc.spi.JdbcConnectionPoolProvider;
+
+/**
+ * Helidon Database Client JDBC.
+ */
+@Feature(value = "Hikari Connection Pool for JDBC DB Client",
```

Comment:
The other module-info use "Database Client" instead of "DB Client" .
How about "Hikari Connection Pool support for Helidon JDBC Database Client"

Reply:
ook